src/modules/lua_check.c

```diff
@@ -430,6 +443,8 @@ noit_lua_set_metric_f(lua_State *L, mtev_boolean allow_whence,
   int64_t __l = 0;
   uint64_t __L = 0;
 
+  mtev_memory_begin();
+
   if(lua_gettop(L) < 2 || lua_gettop(L) > 4) luaL_error(L, "need 2-4 arguments: <metric_name> <value> [whence_s] [whence_us]");
```

Comment:
We can't error inside an SMR transaction b/c it will leave without closing it.

Reply:
I thought this would kill the process

Reply:
No, it will break the runtime just for this check execution.  the top level lua_resume will return an error.

Reply:
Yes, I see, it does a ``_longjmp``